RELEASE_NOTES.md

```diff
@@ -1,5 +1,13 @@
 # Next release
 
+## API changes
+
+PR [#333](https://github.com/IAMconsortium/pyam/pull/333) changes the second input name of `fill_series()` to `time`. It can still be an integer. 
```

Comment:
This seems to be PR #334...?

Reply:
Resolved in the new commit, not sure how you can see the number before making a PR?

Reply:
you can guess it by looking at the issues and PR list (hoping that nobody creates a new issue/PR while you push. Or you first create a PR, then add another commit adding the release-notes update.